### PR TITLE
docs(api): update 'help' and 'version' usage

### PR DESCRIPTION
**Pull request by snitin315**
update `help` and `version` usage

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -210,10 +210,18 @@ This is the lookup priority in increasing order
...
 __List basic commands and flags available on the cli__
 
+Valid syntax for help is `webpack help [command] [option]'` or `'webpack [command] --help`
+
 ```bash
 npx webpack --help
+
+# or
+
+npx webpack help
```
Comment (snitin315) [inline src/content/api/cli.md:224]:
```suggestion
# or
npx webpack help
```

File: src/content/api/cli.md [hunk 2]
```diff
@@ -222,12 +230,24 @@ __List all supported commands and flags by cli__
...
 __Show version of installed packages and sub-packages__
 
 To inspect the version of `webpack` and `webpack-cli` you are using just run command:
 
 ```bash
 npx webpack --version
+
+# or
+
+npx webpack version
```
Comment (snitin315) [inline src/content/api/cli.md:250]:
```suggestion
# or
npx webpack version
```

File: src/content/api/cli.md [hunk 3]
```diff
@@ -210,10 +210,16 @@ This is the lookup priority in increasing order
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
+### `help`
```
Comment (chenxsan) [inline src/content/api/cli.md:213]:
```suggestion
### help
```
Can you rebase the pull request? And let's avoid inline code in heading, it makes the page messy in my opinion.

Comment (snitin315) [reply src/content/api/cli.md]: resolved

File: src/content/api/cli.md [hunk 4]
```diff
@@ -210,10 +210,18 @@ This is the lookup priority in increasing order
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
+### help
+
 **List basic commands and flags available on the cli**
 
+Valid syntax for help is `webpack help [command] [option]'` or `'webpack [command] --help`
```
[the remark this answers is not in the document]
Comment (snitin315) [reply src/content/api/cli.md]: Thanks

**Review by chenxsan — APPROVED** [review]

**Pull request merged**